src/components/Tabs/SettingsPanel/styles.module.css

```
@@ -0,0 +1,121 @@
+.settingsPanel {
+  padding: 24px;
+  background-color: #faf5ff;
+  border-radius: 8px;
+}
+
+.panelTitle {
+  font-size: 18px;
+  font-weight: 600;
+  margin-bottom: 12px;
+  margin: 0 0 12px 0;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)